src/ripple/app/ledger/Ledger.cpp

```diff
@@ -773,7 +773,10 @@ Ledger::walkLedger(beast::Journal j) const
     }
     else
     {
-        stateMap_->walkMap(missingNodes1, 32);
```

Comment:
Generally, using `walkMap` is the wrong thing, but let's not worry about that for this PR.